wireguard-install.sh

```diff
@@ -177,13 +177,15 @@ Address = ${SERVER_WG_IPV4}/24,${SERVER_WG_IPV6}/64
 ListenPort = ${SERVER_PORT}
 PrivateKey = ${SERVER_PRIV_KEY}" >"/etc/wireguard/${SERVER_WG_NIC}.conf"
 
+	# Add firewall rule for Ubuntu 20.04 (others?)
+	# iptables -I INPUT 2 -p udp -m udp --dport ${SERVER_PORT} -j ACCEPT
```

Comment:
This line should be in `else` branch of next `if` statement, because `if pgrep firewalld` detects is `firewalld` or plain iptables are used